platforms/azure/giantnetes/variables.tf

```diff
@@ -1,3 +1,14 @@
+variable "github_token" {
+  type        = string
+  description = "Your personal GITHUB token, used to get access to the private repository 'employees' to get the list of users."
```

Comment:
how will this work for CI?

Reply:
I guess we need to add the variable to conveyor jobs.
Will test that scenario before I merge.

Reply:
Addressed with https://github.com/giantswarm/conveyor/pull/87